assemble/bin/accumulo

```diff
@@ -127,13 +127,12 @@ function main() {
     JAVA=($ACCUMULO_JAVA_PREFIX $JAVA)
   fi
   
-  CLASSPATH="${lib}/accumulo-start.jar:${conf}:${lib}/slf4j-api.jar:${lib}/slf4j-log4j12.jar:${lib}/log4j.jar:${CLASSPATH}"
 
-  exec "${JAVA[@]}" "-Dapp=$ACCUMULO_CMD" \
+  CLASSPATH="${conf}:${lib}/*:${CLASSPATH}"
```

Comment:
The common case would be to have this be non-null, since the rest of the class path will likely be set in the env file. This keeps it simple and readable, and extra colons in the class path don't cause issues.